kokoro/builds/build.sh

```diff
@@ -199,5 +199,5 @@ else
     --isolation=process --storage-opt 'size=50GB' \
     ${CONTAINER} \
     'C:/Program Files/Git/bin/bash.exe' -c \
-    "/c/mnt/github/orbitprofiler/kokoro/builds/build.sh in_docker" | tr -d '\r'
```

Comment:
If the 'tr' is needed for anything you can try to read PIPESTATUS[0] afterwards and return that to kokoro.

Reply:
Yeah, good point. I thought about doing something like that. But the `tr`-solution was never finished anyway (logfiles looked better but there still superfluous line breaks.). So I decided to postpone the whole topic which means logfiles continue to look like they are right now.